docs/guide.rst

```diff
@@ -773,6 +774,45 @@ locations without any change to the resulting `fmf` tree:
 This gives you a nice flexibility to extend the metadata when and
 where needed as your project organically grows.
 
+
+.. _link-issues:
+
+Link Issues
+------------------------------------------------------------------
+
+You can link issues to the test or plan that covers it. This can
+be either done directly during creation of a new test or plan, or
+later on using the ``tmt link`` command:
```

Comment:
```suggestion
You can link issues to the test or plan that covers it. This can
be done either directly during creation of a new test or plan, or
later on, using the ``tmt link`` command:
```
(or just 'later')

Reply:
Ack, let's go with "just later".

Reply:
Covered by 974f8cc6.